app/assets/stylesheets/custom.css.scss

```diff
@@ -227,13 +227,18 @@ p.copyright {
     opacity: 0;
     filter: alpha(opacity=0);
 }
+
 .q-width{
   margin: auto;
   width: 40%;
   min-width: 300px;
 }
-.row .btn{
-  margin-bottom: 1em;
+
+.btn-margin{

```

Comment:
@DimaSmagulov as i know u use .btn-group in admin/users and i think they both (pages) must be similar. But thanks for help